src/main/webapp/js/webrtc_adaptor.js

```diff
@@ -467,6 +467,7 @@ export class WebRTCAdaptor
 				token : token,
 				subscriberId: typeof subscriberId !== undefined ? subscriberId : "" ,
 				subscriberCode: typeof subscriberCode !== undefined ? subscriberCode : "",
+				streamName : streamName,
```

Comment:
I think streamName can be "undefined" here and it may cause the server to name the stream as "undefined"